index.js

```diff
@@ -1,5 +1,5 @@
 // You'll want to set these with either `CLIENT_ID=abc zapier test` or `zapier env 1.0.0 CLIENT_ID abc`
-process.env.BASE_URL = process.env.BASE_URL || 'https://oauth-test.zapier.com/oauth';
+process.env.BASE_URL = process.env.BASE_URL || 'https://auth-json-server.zapier.ninja';
```

Comment:
Shouldn't this use a public host so that users can run the tests?

Reply:
https://auth-json-server.zapier.ninja is a public host.

Reply:
O wow! I thought we reserved `.ninja` to the staging environment only available via VPN. Learned something today :)

Reply:
That was what I thought, too, until devops told me otherwise. :)